### PR TITLE
Adding postalCode settings / provider

### PR DESCRIPTION
**Pull request by jlacivita**

### CODE REVIEW

File: src/modules/device.json [hunk 1]
```diff
@@ -101,6 +101,101 @@
...
+                            "19103"
+                        ]
+                    }
+                },
+                {
+                    "name": "capabilities",
+                    "x-provides": "xrn:firebolt:capability:settings:device"
+                }
+            ],
+            "summary": "Request the user-asigned postal code of this device from the device settings provider.",
```
Comment (kpears201) [inline src/modules/device.json:121]: typo: `assigned`

File: src/modules/device.json [hunk 2]
```diff
@@ -101,6 +101,101 @@
...
+                    "name": "property"
+                },
+                {
+                    "name": "capabilities",
+                    "x-manages": [
+                        "xrn:firebolt:capability:settings:device"
+                    ]
+                }
+            ],
+            "summary": "The user-asigned postal code of this device.",
```
Comment (kpears201) [inline src/modules/device.json:178]: typo: `assigned`

File: src/modules/device.json (hunk 1 above)
Comment (jlacivita) [reply src/modules/device.json:121]: fixed

**Review by kpears201 — APPROVED** [review]

**Pull request merged**